package.json

```
@@ -2,24 +2,35 @@
   "name": "network-checker",
   "version": "0.1.1",
   "description": "A module for checking/monitoring a URL",
-  "main": "index.js",
-  "typings": "index.d.ts",
+  "main": "dist/index.js",
   "scripts": {
-    "prepublish": "coffee -c index.coffee",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prettify": "balena-lint --fix lib/ test/",
+    "lint": "balena-lint lib/ test/ && tsc --noEmit",
+    "clean": "rimraf dist",
+    "build": "npm run clean && tsc --project ./",
```

The default will be the tsconfig.json so you can just run tsc without args:
```suggestion
    "build": "npm run clean && tsc",
```

---

Build now runs with tsconfig.release.json! [ref](https://github.com/balena-io-modules/network-checker/pull/9/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R10)